services/src/main/java/org/keycloak/email/DefaultEmailSenderProvider.java

```diff
@@ -97,11 +100,8 @@ public void send(Map<String, String> config, String address, String subject, Str
                 props.setProperty("mail.smtp.starttls.enable", "true");
             }
 
-            if (ssl || starttls) {
-                props.put("mail.smtp.ssl.protocols", SUPPORTED_SSL_PROTOCOLS);
-
-                setupTruststore(props);
-            }
+            props.put("mail.smtp.ssl.protocols", SUPPORTED_SSL_PROTOCOLS);
```

Comment:
Is there a possiblity that ssl will be used if all of these properties all false?

Reply:
Reverted change.
Yes (with dane - https://www.rfc-editor.org/rfc/rfc6698) , but jakarta doesn't support it anyway.